lisa/features/disks.py

```diff
@@ -19,6 +19,9 @@ def can_disable(cls) -> bool:
     def enabled(self) -> bool:
         return True
 
+    def get_raw_data_disks(self) -> List[str]:
```

Comment:
How about `get_physical_disks`? I think it includes the os disk too. And I'm not sure the raw disk is a right term. The raw disk may indicate a disk, which is not formatted. Once it's formatted, it's not a raw disk.

Reply:
here it will get all data disks name, /dev/sdc etc

Reply:
So, the method name shouldn't contain `data`